frontend/src/pages/modelRegistry/ModelRegistryRoutes.tsx

```diff
@@ -21,6 +22,7 @@ const ModelRegistryRoutes: React.FC = () => (
         />
       }
     >
+      <Route path="registerModel" element={<RegisterModel />} />
```

Comment:
The route should be after the index route, right before the `registeredModels/*` route.